### PR TITLE
Refactor Test to use API Call

### PR DESCRIPTION
**Pull request by diogormendes**
### Status
PENDING CI / PENDING REVIEW

### What does this PR accomplish?
- Reproduces #20442 

### How to test this manually?
- `yarn test-cypress-open`
- `frontend/test/metabase/scenarios/admin/settings/sso/google.cy.spec.js`

> _(For a fixed bug)_
- The bug was fixed in #20462

### Screenshots:

Screen Shot 2022-02-14 at 08 46 18

### CODE REVIEW

**Comment by diogormendes** [comment]
@nemanjaglumac , address the second test using API as you mentioned. Thanks for the tip. Also removed the `saveSettings();` in favor to create a new `successfullySaveSettings()` function as well mentioned by you.

Added a `.clear()` before typing again on the first test since it was typing twice the same value.

**Comment by nemanjaglumac** [comment]
The merge conflict you're seeing is because aafc68ab1198ba26c6c30a8d609409f7f8136231 was already merged. No need to include the same changes again. Only the diff.

File: frontend/test/metabase/scenarios/admin/settings/sso/google.cy.spec.js [hunk 1]
```diff
@@ -11,37 +11,34 @@ describe("scenarios > admin > settings > SSO > Google", () => {
...
+    cy.button("Save changes").click();
     cy.findByText(
       'Invalid Google Sign-In Client ID: must end with ".apps.googleusercontent.com"',
     );
   });
 });
 
-function saveSettings() {
+function successfullySaveSettings() {
   cy.button("Save changes").click();
```
Comment (nemanjaglumac) [inline frontend/test/metabase/scenarios/admin/settings/sso/google.cy.spec.js:43]: Don't we need `cy.findByText("Success");` back here?

**Comment by nemanjaglumac** [comment]
> Added a `.clear()` before typing again on the first test since it was typing twice the same value.

It wasn't the same value. `id`s were different, but adding `clear()` was a good call anyway. The only thing that matters for that test is that it can save again.

**Review by nemanjaglumac — APPROVED** [review]
Thank you for updating the test and making it faster.

**Pull request merged**